addons/knobs/src/components/types/Color.tsx

```diff
@@ -1,12 +1,45 @@
+// @ts-ignore
```

Comment:
You can create a `typings.d.ts` file in `src` folder with: 
```ts
declare module 'global';
```


Then you will be able to remove these `// @ts-ignore` without getting TS error back.